auth-api/src/auth_api/services/google_store.py

```diff
@@ -0,0 +1,89 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for interacting with Google Cloud Storage (GCS)."""
+
+import uuid
+from datetime import timedelta
+
+from flask import current_app
+from google.cloud import storage
+from structured_logging import StructuredLogging
+
+from auth_api.utils.constants import AFFIDAVIT_FOLDER_NAME
+
+logger = StructuredLogging.get_logger()
+
+
+class GoogleStoreService:
```

Comment:
Isn't this duplicated here? queue_services/account-mailer/src/account_mailer/services/google_store.py

Reply:
potentially, but they are using different methods, auth api is only doing signed get/put urls, while mailer is only doing donwload (and upload is used in tests). but then we might as well put it in the common components at some point